src/core/eventqueue.c

```diff
@@ -22,6 +22,8 @@
  * 02110-1301, USA.  */
 
 #include "eventqueue.h"
+#include <glib-object.h>
+#include <gobject/gvaluecollector.h>
```

Comment:
Why this include?

Reply:
```
core/eventqueue.c:73:48: warning: implicit declaration of function ‘G_CALLBACK’ [-Wimplicit-function-declaration]
   73 |   g_source_set_callback (source, (GSourceFunc) G_CALLBACK (func), data, NULL);
```

Reply:
I was asking only about `#include <gobject/gvaluecollector.h>`. This seems to be not needed...

Reply:
P.S. At least in `metacity` this was unused:
https://gitlab.gnome.org/GNOME/metacity/commit/abdd2bf2713f844b99e7e96bbdebcec9be4fee87

Reply:
I removed `#include <gobject/gvaluecollector.h>`, eventqueue removal should be done in different PR if necessary.